src/MSBuild/CommandLineSwitches.cs

```diff
@@ -277,7 +277,7 @@ private struct ParameterizedSwitchInfo
             new ParameterizedSwitchInfo(  new string[] { "warnnotaserror", "noerr" },           ParameterizedSwitch.WarningsNotAsErrors,        null,                           true,           "MissingWarnNotAsErrorParameterError", true,   false,   "HelpMessage_40_WarnNotAsErrorSwitch"),
             new ParameterizedSwitchInfo(  new string[] { "warnasmessage", "nowarn" },           ParameterizedSwitch.WarningsAsMessages,         null,                           true,           "MissingWarnAsMessageParameterError",  true,   false,   "HelpMessage_29_WarnAsMessageSwitch"),
             new ParameterizedSwitchInfo(  new string[] { "binarylogger", "bl" },                ParameterizedSwitch.BinaryLogger,               null,                           false,          null,                                  true,   false,   "HelpMessage_30_BinaryLoggerSwitch"),
-            new ParameterizedSwitchInfo(  new string[] { "analyze", },                          ParameterizedSwitch.Analyze,                    null,                           false,          null,                                  true,   false,   null),
+            new ParameterizedSwitchInfo(  new string[] { "check", },                            ParameterizedSwitch.Check,                      null,                           false,          null,                                  true,   false,   null),
```

Comment:
I think we should rename the switch to "buildCheck".  "check" is a little bit abstract. For comparison:
```
msbuild -buildCheck
```
```
msbuild -check
```
I think naming infrastructure classes and other things "Check" is fine because the namespace is "BuildCheck" but not for the switch. However, I don't feel very strongly about this. @JanKrivanek, what do you think?

Reply:
I do not have strong opinion.
There was no strong opposition against just `-check` yesterday - so I'm fine leaving as is.